tools/rvn/CommandLineApp.cs

```diff
@@ -51,6 +56,114 @@ public static int Run(string[] args)
             }
         }
 
+        private static void ConfigureSetupPackage()
+        {
+            _app.Command("create-setup-package", cmd =>
```

Comment:
It would be awesome to have a test making a zip and checking all things are there as we want them to. We'd need to fake the DNS change and LE API integration.

Reply:
Also please think of other pieces that we could test.